### PR TITLE
Fix ssw-py issues

### PR DESCRIPTION
**Pull request by inumanag**
Hi @rawagiha 

Thank you for the indelpost---really nice piece of software!

We are extensively using indelpost in our pharmacogenomics suite Aldy. However, we have received lots of bug reports regarding the ssw-py dependency. ssw-py is quite hard to install, does not support newer Python versions, and is not compatible with ARM/M1 machines. 

Here I integrated ssw-py directly into the indelpost tree and cleaned up various build issues with. ARM/M1 is also supported. We are using this version internally for now, and it would be great if you are able to merge it into the master branch.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,70 +1,81 @@
...
-
-    subprocess.check_call(
-        ["python", "-m", "pip", "install", pkg_name], stdout=subprocess.DEVNULL
-    )
-
-
-try:
-    from Cython.Build import cythonize
-    from Cython.Distutils import build_ext
-except ImportError:
```
Comment (inumanag) [inline setup.py:15]: pyproject.toml should take care of these now, I guess

File: setup.py [hunk 2]
```diff
@@ -1,70 +1,81 @@
...
-    from Cython.Distutils import build_ext
-
-try:
-    from pysam import get_include as pysam_get_include
-except ImportError:
-    pip_install("pysam")
-    from pysam import get_include as pysam_get_include
-
-try:
-    import numpy
```
Comment (inumanag) [inline setup.py:28]: It seems that numpy is not used during the setup. Removing it as it can occasionally cause issues.

**Review by rawagiha — APPROVED** [review]

**Comment by rawagiha** [comment]
@ inumanag Thank you so much for your contribution. This is really helpful to alleviate the cumbersome installation issue.

File: setup.py (hunk 1 above)
Comment (rawagiha) [reply setup.py:15]: Unfortunately, pyproject.toml does not take care of (hence, doc failling). 
If this deleted part is recovered, will it affect your fix?

Comment (inumanag) [reply setup.py:15]: It won't affect the fix--- please keep it in that case.

**Pull request merged**